docroot/sites/its.uiowa.edu/modules/its_core/its_core.module

```diff
@@ -210,3 +207,32 @@ function its_core_preprocess_views_view_unformatted(&$variables) {
 
   }
 }
+
+/**
+ * Implements hook_preprocess_breadcrumb().
+ */
+function its_core_preprocess_breadcrumb(&$variables): void {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if (!isset($node) || $node->getType() !== 'service') {
+    return;
+  }
+  // Grab our referenced Service Category taxonomy term
+  // and add a crumb to the breadcrumb.
+  if ($node->hasField('field_service_category')) {
+    $category = $node->field_service_category?->entity;
+    $category_name = $category?->name?->value;
+    $category_path = \Drupal::service('path_alias.manager')
+      ->getAliasByPath('/taxonomy/term/' . $category?->tid?->value);
+    if (isset($category_name) && isset($category_path)) {
+      $variables['breadcrumb'][] = [
+        'text' => $category_name,
+        'url' => $category_path,
+      ];
+    }
+  }
+  // Also include an unlinked crumb to the current page
+  // matching our breadcrumb pattern.
+  $variables['breadcrumb'][] = [
+    'text' => $node->getTitle(),
+  ];
```

Comment:
I'm fine with removing this, matching non-menu-based breadcrumbs.

```suggestion
```

Reply:
@briand44 are you okay with this based on https://github.com/uiowa/uiowa/pull/6984#issuecomment-1756150733 ?

Reply:
umm... I would like to understand a little more. Will there be other examples of breadcrumbs on ITS without the node title?

Reply:
articles for one: https://sandbox.prod.drupal.uiowa.edu/news/2023/02/joes-new-article-test

unless it lives in the menu system, it doesn't get the content title appended. i imagine support articles, software be the same unless custom breadcrumb work will be done there too.

Reply:
Articles: 
<img width="280" alt="image" src="https://github.com/uiowa/uiowa/assets/49074566/6a51199f-eb30-492b-8712-7eed2afe97c8">

And and pages that aren't placed directly in the menu:
<img width="292" alt="image" src="https://github.com/uiowa/uiowa/assets/49074566/729bf9ee-6ee6-46f4-a33f-e7d94aff75ee">

Pages that are placed in the menu get the page title added to the breadcrumb as part of that
<img width="650" alt="image" src="https://github.com/uiowa/uiowa/assets/49074566/5ba7cb28-ee60-42df-a2ce-46aace5614e4">

Reply:
Thanks, yeah I think that makes sense then.